tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
+# using the :func:`mne.combine_evoked` function. A simple difference can be
+# generated by negating one of the :class:`~mne.Evoked` objects passed into the
+# function. We'll then plot the difference wave at each sensor using
+# :meth:`~mne.Evoked.plot_topo`:
 
-from mne.minimum_norm import apply_inverse_raw  # noqa
-start, stop = raw.time_as_index([0, 15])  # read the first 15s of data
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
+evoked_diff = mne.combine_evoked([aud_evoked, -vis_evoked], weights='equal')
+evoked_diff.pick_types('mag').plot_topo(color='r', legend=False)
 
 ##############################################################################
-# Save result in stc files:
+# Time-frequency analysis
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The :mod:`mne.time_frequency` submodule provides implementations of several
+# algorithms to compute time-frequency representations, power spectral density,
+# and cross-spectral density. Here, for example, we'll compute for the
+# auditory epochs the induced power at different frequencies and times, using
+# Morlet wavelets:
 
-stc.save('mne_dSPM_raw_inverse_Aud')
+frequencies = np.arange(7, 30, 3)
+power = mne.time_frequency.tfr_morlet(aud_epochs, n_cycles=2, return_itc=False,
+                                      freqs=frequencies, decim=3)
+power.plot(['MEG 1332'])
 
 ##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
-#
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
+# Inverse modeling
+# ^^^^^^^^^^^^^^^^
 #
-# Want to know more ?
-# ^^^^^^^^^^^^^^^^^^^
+# Finally, we can estimate the cortical origins of the evoked activity by
+# projecting the sensor data into this subject's :term:`source space`.
+# MNE-Python supports lots of ways of doing this (minimum-norm estimation,
+# dipole fitting, beamformers, etc); here we'll use minimum-norm estimation
+# (MNE) to generate a continuous cortical activation map. To do this we'll
+# need the inverse operator for this subject (the sample data includes one
+# that's been pre-computed), and we'll need to specify a regularization
+# parameter:
```

Comment:
I would add "inverse operator" in glossary and point to it. I would also point to the source estimates entry in glossary.

I would link to sample data entry in doc again.

basically see you can add links to the rest of the doc from here as this can lead people to more materials.

Reply:
+1 for copious linking here, keeping this 'Overview' at a bare minimum.

@drammock I see you have plans for source estimation in #5944 I'd be happy to follow your lead in revising some relevant 'tutorials', such as (link now, fix later?)

auto_tutorials/source-modeling/plot_forward.html
auto_tutorials/source-modeling/plot_mne_dspm_source_localization.html
auto_tutorials/source-modeling/plot_mne_solutions.html

This is tightly linked to #5937: I think it would make a lot of sense to land on a concept map of source estimation, then develop a tutorial for each of those.

Reply:
Suggestion for wording:

~To do this we’ll need the inverse operator for this subject (the sample data includes one that’s been pre-computed), and we’ll need to specify a regularization parameter:~

MNE is based on a linear "inverse" operator, which projects M/EEG measurements into a "source space" [REF tutorial?] inside the head (_e.g._, the gray matter surface for cortically constrained estimates). It is based on a full "forward model" [REF], including a definition of the conductivity distribution of the head (REF?: plot_forward.html), as well as the geometry of measurement sensor shape (_i.e._, different MEG sensor types) and their location relative to the head [REF coregistration]. The `sample` data contains pre-computed inverse operator, which we can load from disk for the present examples. In addition, due to the ill-posed nature of inverse problems, we must regularise [REF] the estimation problem in a signal-to-noise-ratio (SNR)-dependent manner.